docs/contribute/contribute-docs.md

```diff
@@ -0,0 +1,229 @@
+# Contributing to ROCm documentation
+
+AMD values and encourages contributions to our code and documentation. If you choose to
```

Comment:
We can reduce the focus of this doc as the new contribution guide covers the first paragraph.

Reply:
it's really just the first sentence and I think it's an okay way to lead into the content...

Reply:
Right, I liked your first sentence so used it in the new Contributing.md.  It's a very welcoming opener. I think you can keep it here too.